Compile failure on 0.5.2 (windows rustc 1.32.0 (9fda7c223 2019-01-16))

```
error[E0308]: mismatched types
  --> C:\Users\jkolb\.cargo\registry\src\github.com-1ecc6299db9ec823\insta-0.5.2\src\runtime.rs:32:51
   |
32 |     path.as_ref().to_str().unwrap().replace('\\', '/').into()
   |                                                   ^^^ expected &str, found char
   |
   = note: expected type `&str`
              found type `char`

error: aborting due to previous error
```

--- a/src/runtime.rs
+++ b/src/runtime.rs
@@ -29,7 +29,7 @@ enum UpdateBehavior {
 
 #[cfg(windows)]
 fn path_to_storage<P: AsRef<Path>>(path: P) -> String {
-    path.as_ref().to_str().unwrap().replace('\\', '/').into()
+    path.as_ref().to_str().unwrap().replace('\\', "/").into()
 }
 
 #[cfg(not(windows))]

